src/services/ldap/strategies/general.js

```diff
@@ -28,13 +28,9 @@ class GeneralLDAPStrategy extends AbstractLDAPStrategy {
 	 * (Array) roles = ['teacher', 'student', 'administrator']
 	 * @memberof GeneralLDAPStrategy
 	 */
-	async getUsers(school) {
-		const {
-			userAttributeNameMapping,
-			userPathAdditions,
-			roleType,
-			roleAttributeNameMapping,
-		} = this.config.providerOptions;
+	async getUsers() {
```

Comment:
that would be correct, however the interface describes the method with school param...

Reply:
From a functional point of view, it should not make a difference. Since in JS extra arguments can be passed to a function, this method still fulfills the interface.